packages/react-next/src/components/Link/Link.tsx

```diff
@@ -1,40 +1,10 @@
-import * as React from 'react';
-import { getGlobalClassNames, ITheme } from '../../Styling';
-import { memoizeFunction, styled } from '../../Utilities';
-import { LinkBase } from './Link.base';
-import { ILinkProps, ILinkStyleProps, ILinkStyles } from './Link.types';
+import { compose } from '@fluentui/react-compose';
+import { LinkBase } from './LinkBase';
 import * as classes from './Link.scss';
-import { css } from 'office-ui-fabric-react/lib/Utilities';
+import { LinkProps } from './Link.types';
 
-const GlobalClassNames = {
-  root: 'ms-Link',
-};
-
-const getStaticStylesMemoized = memoizeFunction(
-  (theme: ITheme, className?: string, isButton?: boolean, isDisabled?: boolean) => {
-    const globalClassNames = getGlobalClassNames(GlobalClassNames, theme);
```

Comment:
you are also removing some static class names

Reply:
Yes, I'm still trying to figure out how to completely account for all of this.

Reply:
i think we need to take a smaller step. use compose without removing `styled`

Reply:
Resolved by reverting to use `styled`.